docs/pages/versions/v52.0.0/config/package-json.mdx

```diff
@@ -33,3 +33,55 @@ By specifying the library under the `install.exclude` array in the **package.jso
 Allows configuring module resolution behavior by using `autolinking` property in **package.json**.
 
 For complete reference, see [Autolinking configuration](/modules/autolinking/#configuration).
+
+## `doctor`
+
+Allows configuring the behavior of the [`npx expo-doctor`](/develop/tools/#expo-doctor) command.
+
+### `reactNativeDirectoryCheck`
+
+By default, Expo Doctor validates your project's packages against the [React Native directory](https://reactnative.directory/). This check throws a warning with a list of packages that are not included in the React Native Directory.
+
+You can customize this check by adding the following configuration in your project's **package.json** file:
+
+```json package.json
+{
+  "expo": {
+    "doctor": {
+      "reactNativeDirectoryCheck": {
+        /* @info Use this option to disable/enable the check. */
+        "enabled": true,
+        /* @end */
+        /* @info Use this option to exclude specific packages. */
+        "exclude": ["/foo/", "bar"],
+        /* @end */
+        /* @info Use this option to disable/enable listing unknown packages. */
+        "listUnknownPackages": true
+        /* @end */
+      }
+    }
+  }
+}
+```
+
+By default, the check is enabled and unknown packages are listed.
+
+### `appConfigFieldsNotSyncedCheck`
+
+Expo Doctor checks if your project includes native project folders such as **android** or **ios**. If these folders exist but are not listed in your **.gitignore** or [**.easignore**](/build-reference/easignore) files, Expo Doctor verifies the presence of an **app.json** or **app.config.js** file. If this file exists, it means your project is configured to use [Prebuild](/workflow/prebuild).
```

Comment:
```suggestion
Expo Doctor checks if your project includes native project directories such as **android** or **ios**. If these directories exist but are not listed in your **.gitignore** or [**.easignore**](/build-reference/easignore) files, Expo Doctor verifies the presence of an app config file. If this file exists, it means your project is configured to use [Prebuild](/workflow/prebuild).
```

app.config.ts is also the third variant of app config files that people sometimes have in their project instead of other two files. Let's just call them app config files instead of listing all the variants here.

Also, we prefer using directory over folder for consistency.